### DISCUSSION TITLE
ESP32 & CMT2300A --> capacitor needed?

### DISCUSSION THREAD

**Discussion by wg25** [tbnobody/OpenDTU#2183] [Q&A]
Maybe I didn't catch the point correct but do I need also an additional capacitor when using the CMT2300A for the HMS series? I can't find it anywhere... And if so, which size and where to install?

**Comment by DerGreifer** [comment]
Hello!
What kind of usb power supply are you using? 5V /1A or ? I recommend to place the capacitor near the 3v3 and the ground pins! No matter if you place it near the ESP32, the NRF2401 or the cmt2300A. It depends of the wiring and space on your pcb. 100 uF (microfarad) and minimum 6,3V or 10V are ok! It will also work mostly with only 47uF...depends on the rf power of your NRF2401 or CMT2300A modules and the stability of the power supply!
Hope this will help!
